client/build.gradle

```diff
@@ -18,7 +18,7 @@ configurations.all {
 
 dependencies {
     implementation project(':conductor-common')
-
+    implementation project(':conductor-core')
```

Comment:
This dependency seems unnecessary and would bring in a large number of libraries into any application that would use the conductor client.